ui/components/config/provider-form.tsx

```
@@ -591,7 +591,7 @@ export default function ProviderForm({ provider, onSave, onCancel, existingProvi
                             </div>
                             <div>
                               <div className="flex items-center gap-2">
-                                <label className="text-sm font-medium">Models (Optional)</label>
+                                <label className="text-sm font-medium">Models</label>
```

Biome (2.1.2)

[error] A form label must be associated with an input.

Consider adding a `for` or `htmlFor` attribute to the label element or moving the input element to inside the label element.

(lint/a11y/noLabelWithoutControl)